### PR TITLE
delete minor unreachable code caused by log.Fatal

### PR DESCRIPTION
**Pull request by Abirdcfly**
Signed-off-by: Abirdcfly <fp544037857@gmail.com>

**Description**

Changes proposed in this pull request:

- delete minor unreachable code caused by log.Fatal
- bump Application Connector test image

**Related issue(s)**

See also #15061 @mvshao

### CODE REVIEW

File: tests/components/application-connector/resources/charts/gateway-test/values.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ global:
 
     mockApplication:
       name: "mock-app"
-      version: "29da0039"
+      version: "de411ec6"
```
Comment (franpog859) [inline tests/components/application-connector/resources/charts/gateway-test/values.yaml:12]:
```suggestion
      version: "PR-15714"
```

**Review by franpog859 — COMMENTED** [review]
The image version that is built in this PR is `PR-15714`. Please, change this and I think it's ok to merge, right @mvshao ?

**Comment by mvshao** [comment]
Yes, @franpog859 it looks ok, now I will run the pipeline tests.

**Review by mvshao — APPROVED** [review]
LGTM!

**Pull request merged**